src/SFA.DAS.LevyTransferMatching.Web/Views/Shared/_OpportunitySummary.cshtml

```diff
@@ -23,7 +23,7 @@
     @if (!Model.HideFooter)
    {
         <div class="app-highlight__footer">
-            <p class="govuk-heading-s">Total funding available: up to £@Model.Amount.ToString("N0") available in @Model.YearDescription</p>
+            <p class="govuk-heading-s">Total funding available: £@Model.Amount.ToString("N0")</p>
```

Comment:
As above, if this property on the model is no longer used, it should be removed from the model, the mapper and any unit tests.